### ISSUE TITLE
[FR] Unicode barplot

### ISSUE THREAD

**Issue by zsoerenm** [JuliaPlots/Plots.jl#4313]
I'd like to have the beautiful bar plots from Unicodeplots:
![Image](https://raw.githubusercontent.com/JuliaPlots/UnicodePlots.jl/unicodeplots-docs/2.10/barplot1.png)
I tried:
```
using Plots
unicodeplots()
y = randn(6) * 2 .+ 45
bar(1:6, y, legend = false, orientation = :horizontal)
```
and I got:
![image](https://user-images.githubusercontent.com/375486/185053975-76729e0e-cc53-44e2-b88b-0f818621c926.png)
Is something like this already possible?

**Comment by t-bltg** [comment]
I agree with you that the native [`barplot`](https://github.com/JuliaPlots/UnicodePlots.jl#barplot) looks better than the `Plots` recipe, so we should bypass it (as it is currently done e.g. for [`heatmap`](https://github.com/JuliaPlots/Plots.jl/blob/7dcda22e12b4e2dc48e4a025f0338a8402c16d7c/src/backends/unicodeplots.jl#L190),  `spy` or `densityplot`).

I'll have a look when I'll have time (probably in the next few weeks).

**Comment by t-bltg** [comment]
I've had a look, but it seems more harder to implement than expected since `:bar` is a dependency of `:barbins`, ...
Furthermore, `UnicodePlots` only support horizontal bar plots, whilst `Plots` supports both vertical and horizontal ones.
